domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -97,6 +103,40 @@ class StateRetriever @Inject constructor(
     return answerGroups
   }
 
+  // Creates the list of hints and solution objects from JSON
```

Comment:
How does this deal with Solution? Because in code I am seeing only Hints part.

Reply:
I missed that part. I added it now, please check.